src/PE/signature/OIDToString.cpp

```diff
@@ -2246,9 +2246,14 @@ const char* oid_to_string(const oid_t& oid) {
     { "1.3.6.1.4.1.311.10.3.28",          "szOID_PLATFORM_MANIFEST_BINARY_ID" },
     { "1.3.6.1.4.1.311.2.6.1",            "SPC_RELAXED_PE_MARKER_CHECK_OBJID" },
   };
-  auto   it  = oid_to_str.find(oid);
-  return it == oid_to_str.end() ? oid.c_str() : it->second;
+  auto it = oid_to_str.find(oid);
```

Comment:
What is the motivation behind this change?

Reply:
I missed to read your comment before writing that but I still to get what you aim to do with this change

Reply:
Ok I see what went wrong. `oid_to_string` is expecting a `const std::string&` but in your code you (implicitly) create the `std::string` in during the call. I'll fix that don't worry.